homeassistant/components/sensor/deconz.py

```diff
@@ -158,7 +158,7 @@ def unique_id(self):
     @property
     def device_class(self):
         """Return the class of the sensor."""
-        return self._device_class
+        return DEVICE_CLASS_BATTERY
 
     @property
     def icon(self):
```

Comment:
As long as no override icon is set, the icon should not be used with device class. The device class attribute is supposed to *replace* the icon attribute for most use cases after all.